addon/.eslintrc.js

```diff
@@ -2,17 +2,18 @@
 
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
```

Comment:
I think you'll want to use overrides for ts files instead.

TS doesn't know which files are supposed to be CJS and which are supposed to be ESM, so using this parser fro everything gets warnings on the .eslintrc.js, for example